Translations/WinMerge/Portuguese.po

```diff
@@ -1,8 +1,9 @@
+# Released under the "GNU General Public License"
 #
 # Translators:
+# * Hugo Carvalho <hugokarvalho@hotmail.com>
 # * Nelson Simão
```

Comment:
@hugok79 
Thank you for the translation.
This pull request seems to break the changes of the previous Portuguese translation 
https://github.com/WinMerge/winmerge/pull/234/commits/59217418df054979be72d6f164a779d74415d994
Could you retry to create a pull request?
BTW, There is your name in the file contributers.txt of the latest beta release 2.16.5.